website/docs/blueprints/data-analytics/spark-eks-ipv6.md

```diff
@@ -0,0 +1,153 @@
+kube-prometheus-stack-grafana-9f5c9d8fc-zgn98            2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::a
+kube-prometheus-stack-kube-state-metrics-98c74d866-56275 2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::9
+kube-prometheus-stack-operator-67df8bc57d-2d8jh          2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::b
+kube-prometheus-stack-prometheus-node-exporter-5qrqs     2600:1f13:520:1303:c87:4a71:b9ea:417c   2600:1f13:520:1303:c87:4a71:b9ea:417c
+kube-prometheus-stack-prometheus-node-exporter-hcpvk     2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:15b2:b8a3:7f63:cbfa
+kube-prometheus-stack-prometheus-node-exporter-ztkdm     2600:1f13:520:1305:5ee5:b994:c0c2:e4da  2600:1f13:520:1305:5ee5:b994:c0c2:e4da
+prometheus-kube-prometheus-stack-prometheus-0            2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::7
+spark-history-server-6c9f9d7cc4-xzj4c                    2600:1f13:520:1305:5ee5:b994:c0c2:e4da  2600:1f13:520:1305:64b::1
+spark-operator-84c6b48ffc-z2glj                          2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::5
+spark-operator-webhook-init-kbl4s                        2600:1f13:520:1305:5ee5:b994:c0c2:e4da  2600:1f13:520:1305:64b::2
+yunikorn-admission-controller-d675f89c5-f2p47            2600:1f13:520:1303:c87:4a71:b9ea:417c   2600:1f13:520:1303:c87:4a71:b9ea:417c
+yunikorn-scheduler-59d6879975-2rh4d                      2600:1f13:520:1304:15b2:b8a3:7f63:cbfa  2600:1f13:520:1304:a79b::4
+....
+```
+
+### Execute Sample Spark job with Karpenter
+
+Navigate to example directory and submit the Spark job.
+
+```bash
+cd ${DOEKS_HOME}/analytics/terraform/spark-eks-ipv6/examples/karpenter
+kubectl apply -f pyspark-pi-job.yaml
+```
+
+Monitor the job status using the below command. You should see the new nodes triggered by the Karpenter.
+
+```bash
+kubectl get pods -n spark-team-a -w
+```
+
+### Apache YuniKorn Gang Scheduling with NVMe based SSD disk for shuffle storage
+
+Gang Scheduling Spark jobs using Apache YuniKorn and Spark Operator
+
+```bash
+cd ${DOEKS_HOME}/analytics/terraform/spark-eks-ipv6/examples/karpenter/nvme-yunikorn-gang-scheduling
+```
+
+Run the `taxi-trip-execute.sh` script with the following input. You will use the `S3_BUCKET` variable created earlier. Additionally, you must change YOUR_REGION_HERE with the region of your choice, us-west-2 for example.
+
+This script will download some example taxi trip data and create duplicates of it in order to increase the size a bit. This will take a bit of time and will require a relatively fast internet connection.
+
+```bash
+${DOEKS_HOME}/analytics/scripts/taxi-trip-execute.sh ${S3_BUCKET} YOUR_REGION_HERE
+```
+
+Once our sample data is uploaded you can run the Spark job. You will need to replace the <S3_BUCKET> placeholders in this file with the name of the bucket created earlier. You can get that value by running echo $S3_BUCKET.
```

Comment:
```suggestion
Once our sample data is uploaded you can run the Spark job. You will need to replace the `<S3_BUCKET>` placeholders in this file with the name of the bucket created earlier. You can get that value by running echo $S3_BUCKET.
```

These < > chars are breaking the doc compilation

Reply:
Fixed